### PR TITLE
EPMLSTRTAI-766 README changed

### PR DESCRIPTION
**Pull request by venedikttsulenev**
Added synopsis, technologies used, download and run instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,175 @@
...
+1) After downloading you can open project in IntelliJ IDEA. 
+2) Select "maven projects"->"plugins"->"tomcat7".
+3) Execute "run-war" target. Wait for IDEA to build and deploy .war package. 
+4) Open browser and navigate to "localhost:8080/cucumber/swagger-ui.html#/"
+
+##What technologies we used
+
+### [Spring](https://spring.io/docs)
+* [Spring MVC](https://docs.spring.io/spring/docs/current/spring-framework-reference/web.html)
+Spring Web MVC is the web framework that provides architecture of pattern Model-View-Controller.
```
Comment (pomkine) [inline README.md]: I think putting here just a link would be just ok. No need to write a detailed description.

File: README.md [hunk 2]
```diff
@@ -7,13 +7,175 @@
...
+Jackson is JSON library for Java applications. It is a suite of data-processing tools for Java, 
+including the flagship streaming JSON parser / generator library, matching data-binding library 
+(POJOs to and from JSON) and additional data format modules to process data encoded in CSV, (Java) 
+Properties, XML and etc. 
+
+Usage in project:
+
+    Work with Json files
+
+### [Dozer](http://dozer.sourceforge.net/documentation/about.html)
```
Comment (pomkine) [inline README.md]: We are not using Dozer anymore

File: README.md [hunk 3]
```diff
@@ -7,13 +7,175 @@
...
+* Use for unit testing.
+
+*More information about Mockito you can read [here](http://site.mockito.org).*
+
+### [JUnit](http://junit.org/junit5/)
+JUnit is a test framework which uses annotations to identify methods that specify a test.
+
+**Usage in project**
+
+* Use for unit testing.
```
Comment (pomkine) [inline README.md]: That's obvious :)

File: README.md (hunk 2 above)
Comment (venedikttsulenev) [reply README.md]: Removed

File: README.md (hunk 3 above)
Comment (venedikttsulenev) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (venedikttsulenev) [reply README.md]: Shortened

**Pull request merged**